concepts/interfacing-with-c-types/introduction.md

```diff
@@ -103,10 +104,10 @@ Arrays are passed to, and returned from, functions as pointers to their first el
 
 ```c
 int64_t example_arr[] = {1, 2, 3}; // this is an array of 3 signed 8-byte integers
-fn(example_arr); // this is a function that passes a pointer to the beginning of the array as argument
+fn(example_arr); // this is a function that passes a pointer (int64_t *) to the beginning of the array as argument
 ```
 
-Each element in an array has the size of the type pointed by the array.
+Each element in an array has the size of the type pointed by this array pointer.
```

Comment:
Awkward writing near "of the type pointed by this array pointer.  Can we say "referenced by"?  Not sure, something just seems "off" here.